chrome_extension/src/scripts/google_photos_content.ts

```diff
@@ -50,16 +50,31 @@ function handleDeletePhoto(
       return;
     }
 
+    var isBurst = false;
     try {
-      const confirmButton = await waitForElement("[jsshadow] [autofocus]");
+      const confirmButton = await waitForElement("[jsshadow] [autofocus]", 2500);
       confirmButton.click();
     } catch (error) {
-      chrome.runtime.sendMessage({
-        ...resultMessage,
-        success: false,
-        error: "Confirm button not found",
-      });
-      return;
+      // Check if it's a burst photo
+      try {
+        const burstDeleteButton = await waitForElement("[jsshadow] [aria-label='Current photo only']", 2000);
```

Comment:
Thanks for the PR!

So far I've been able to avoid usage of strings like this so it'll work in languages besides English. I'd like to see if there's a unique selector that can be used without the label.

Reply:
The wait here also adds a 2s delay for all non-burst photos. I'm going to play around with this and see if that's avoidable.

Reply:
Good points. We can amend the selector to this to avoid breaking non-English locales:
`document.querySelectorAll("[jsshadow][role='menu'] [role='menuitem']")[1]`

As for delay, I just shortened it to 2000ms, but we can definitely keep the default one.

Reply:
Since we use `querySelector` function (not 'querySelectorAll`) proper selector would be:
`document.querySelector("[jsshadow][role='menu'] [role='menuitem']:nth-of-type(2)")`